src/lib/markbind/src/parser.js

```diff
@@ -211,6 +211,9 @@ class Parser {
 
     $('img, pic, thumbnail').each(function () {
       const elem = $(this);
+      if (elem[0].name === 'thumbnail' && elem.attr('src') === undefined) {
```

Comment:
```suggestion
      if (elem.attr('src') === undefined) {
```

Should we do this then?

Even though pics and imgs must have the src attribute, the author could still forget to include them.

Reply:
Updated